db/external/derby-10.2.2.0-notice.txt

```diff
@@ -0,0 +1,298 @@
+=========================================================================
```

Comment:
Not quite clear to me how the notice should look like - I think this is OK for now, we will learn if there's an issue with it.